ci/bin/lint.dart

```diff
@@ -245,11 +245,15 @@ void main(List<String> arguments) async {
   final ProcessPool pool = ProcessPool();
 
   await for (final WorkerJob job in pool.startWorkers(jobs)) {
-    if (job.result?.stdout.isEmpty ?? true) {
+    if (job.result?.exitCode == 0) {
       continue;
     }
-    print('❌ Failures for ${job.name}:');
-    print(job.result.stdout);
+    if (job.result == null) {
+      print('❗ The clang-tidy job failed to run:\n${job.exception}');
```

Comment:
Should this case break out of the loop, and stop running new jobs?

Reply:
It depends on why it failed to execute: it could be that there was just a file that wasn't found or something. I think it's OK to just print it a bunch of times if it keeps failing.

Reply:
Actually, I suppose it's likely to all fail if one fails, since it's the clang-tidy command that would be not found/not executable.  I'll switch it to break out.